File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/account/CredentialType.java
package com.example.cf_sdk.logic.changebanksdk.model.account;

import com.google.gson.annotations.SerializedName;

/**
 *
 * Credential types.
 */

public enum CredentialType {
    @SerializedName("LOGIN")
    LOGIN,

    @SerializedName("PASSWORD")
    PASSWORD,

    @SerializedName("TEXT")
    TEXT,

    @SerializedName("OPTIONS")
    OPTIONS,

    @SerializedName("IMAGE_DATA")
    IMAGE_DATA,
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/account/CredentialType.kt
package com.example.sdk_no_dagger.changebankapi.model.account

import com.google.gson.annotations.SerializedName

/**
 *
 * Credential types.
 */
enum class CredentialType {
    @SerializedName("LOGIN")
    LOGIN, @SerializedName("PASSWORD")
    PASSWORD, @SerializedName("TEXT")
    TEXT, @SerializedName("OPTIONS")
    OPTIONS, @SerializedName("IMAGE_DATA")
    IMAGE_DATA
}